src/advanced_agent/database/connection.py

```
@@ -0,0 +1,295 @@
+            return {
+                "status": "unhealthy",
+                "error": str(e)
+            }
+    
+    def backup_database(self, backup_path: str) -> bool:
+        """データベースバックアップ"""
+        try:
+            import shutil
+            shutil.copy2(self.db_path, backup_path)
+            logger.info(f"データベースバックアップ完了: {backup_path}")
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)